evm_loader/program/src/evm/buffer.rs

```diff
@@ -20,41 +20,57 @@ enum Inner {
         data: *mut u8,
         range: Range<usize>,
     },
+    ByteVec(Vec<u8>),
```

Comment:
This is the same as:
```
    Owned {
        ptr: NonNull<u8>,
        len: usize,
    },
```

Don't add an additional variant that does the same.

Reply:
Rolled back.